package.json

```diff
@@ -33,10 +33,14 @@
     "karma-commonjs": "^1.0.0",
     "karma-jasmine": "^2.0.1",
     "karma-typescript": "^5.5.2",
-    "typescript": "^4.3.2"
+    "typescript": "^4.3.2",
+    "rxjs": "^6.6.0 || ^7.4.0"
   },
   "bugs": {
     "url": "https://github.com/angelnikolov/ts-cacheable/issues"
   },
-  "homepage": "https://github.com/angelnikolov/ts-cacheable#readme"
+  "homepage": "https://github.com/angelnikolov/ts-cacheable#readme",
+  "dependencies": {
```

Comment:
We had issues when we defined both `devDependencies` and `dependencies` so I'd rather keep them only as `peerDependendices` here. See [this](https://github.com/angelnikolov/ts-cacheable/pull/3)

Reply:
ok, Ill restore that